vignettes/datatable-intro.Rmd

```diff
@@ -16,30 +16,31 @@ knitr::opts_chunk$set(
     error = FALSE,
      tidy = FALSE,
     cache = FALSE,
- collapse = TRUE)
+ collapse = TRUE
+)
 ```
-This vignette introduces the *data.table* syntax, its general form, how to *subset* rows, *select and compute* on columns and perform aggregations *by group*. Familiarity with *data.frame* data structure from base R is useful, but not essential to follow this vignette.
+
+This vignette introduces the `data.table` syntax, its general form, how to *subset* rows, *select and compute* on columns, and perform aggregations *by group*. Familiarity with `data.frame` data structure from base R is useful, but not essential to follow this vignette.
 
 ***
 
-## Data analysis using data.table
+## Data analysis using `data.table`
 
 Data manipulation operations such as *subset*, *group*, *update*, *join* etc., are all inherently related. Keeping these *related operations together* allows for:
 
 * *concise* and *consistent* syntax irrespective of the set of operations you would like to perform to achieve your end goal.
 
-* performing analysis *fluidly* without the cognitive burden of having to map each operation to a particular function from a set of functions available before to perform the analysis.
-
-* *automatically* optimising operations internally, and very effectively, by knowing precisely the data required for each operation and therefore very fast and memory efficient.
+* performing analysis *fluidly* without the cognitive burden of having to map each operation to a particular function from a potentially huge set of functions available before performing the analysis.
 
-Briefly, if you are interested in reducing *programming* and *compute* time tremendously, then this package is for you. The philosophy that *data.table* adheres to makes this possible. Our goal is to illustrate it through this series of vignettes.
+* *automatically* optimising operations internally, and very effectively, by knowing precisely the data required for each operation, leading to very fast and memory efficient code.
 
+Briefly, if you are interested in reducing *programming* and *compute* time tremendously, then this package is for you. The philosophy that `data.table` adheres to makes this possible. Our goal is to illustrate it through this series of vignettes.
```

Comment:
Maybe add something like...

> (See `vignette(package = "data.table")` for the full list.) 

I think most people reading the vignette don't realize it's bundled with the package (and that an online version they're reading might be out of date or otherwise out of sync with the version they have installed...).

Along the same lines, the final line of this vignette refers to the "next" vignette, but I don't know how the reader would know which vignette that is if they're not on the wiki (which has a numbered list).